datarequest.py

```diff
@@ -573,24 +573,40 @@ def api_datarequest_datamanager_review_get(ctx, request_id):
 
 @api.make()
 def api_datarequest_is_owner(ctx, request_id, user_name):
-    result = datarequest_is_owner(ctx, request_id, user_name)
+    """Check if the invoking user is also the owner of a given data request
+
+        This function is a wrapper for datarequest_is_owner.
+
+       :param request_id: Unique identifier of the data request
+       :type request_id: str
+       :param user_name: Username of the user whose ownership is checked
+       :type user_name: str
+
+       :return: `True` if ``user_name`` matches that of the owner of the data request with id ``request_id``, `False` otherwise
+       :rtype: bool
+    """
+
+    is_owner = False
 
-    if not (result['status'] == 0):
-        return api.Error("error", "Something went wrong in determining datarequest ownership")
+    try:
+        is_owner = datarequest_is_owner(ctx, request_id, user_name)
+    except Exception as e:
```

Comment:
Better to raise and catch an error.UUError. Otherwise other bugs are also catched by this Exception.

When another Exception occurs it will be catched higher in de API layer.

Reply:
And if you raise the Exception with a message you can use it here as e.message. If you don't us it you can rmove "as e".

Reply:
Updated: now creating error.UUError instead of generic Exception.